### PR TITLE
clojure update leiningen to v2.5.1

### PR DESCRIPTION
**Pull request by Quantisan**

### CODE REVIEW

File: library/clojure [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # maintainer: Paul Lam <paul@quantisan.com> (@Quantisan)
 
-latest: git://github.com/Quantisan/docker-clojure@27d0141c22bedbf78b908bf5bebd6629fc10a491
+latest: git://github.com/Quantisan/docker-clojure@5c988033c6510c7d91a5f6c7a91b4f63f8f90768
 
+lein-2.5.1: git://github.com/Quantisan/docker-clojure@5c988033c6510c7d91a5f6c7a91b4f63f8f90768
 lein-2.5.0: git://github.com/Quantisan/docker-clojure@27d0141c22bedbf78b908bf5bebd6629fc10a491
```
Comment (tianon) [inline library/clojure]: You don't need to keep 2.5.0 in here, do you? (I mean, you don't plan to update it later if 2.5.0 has a problem -- you'd just update 2.5.1, right?)

We won't delete the tag on the Hub, but then it'll be removed from the "Supported" list on the main image description.

**Comment by tianon** [comment]
LGTM pending discussion of 2.5.0 tag :+1:

Build test of #436; b7b587113dd7749edf36b3411ac82b096aa5f661 (`clojure`):

```console
$ bashbrew build https://raw.githubusercontent.com/docker-library/official-images/b7b587113dd7749edf36b3411ac82b096aa5f661/library/clojure
Cloning clojure (git://github.com/Quantisan/docker-clojure) ...
Processing clojure:latest ...
Processing clojure:lein-2.5.1 ...
Processing clojure:lein-2.5.0 ...
Processing clojure:lein-2.4.3 ...
```

**Comment by Quantisan** [comment]
@tianon removing 2.5.0 in description sounds good. there's nothing that i need to do right?

**Comment by tianon** [comment]
The tags in the description are generated from this file -- think of this file as the group of tags you officially support.

See https://github.com/docker-library/docs/blob/master/generate-dockerfile-links-partial.sh for the code that scrapes this file to update the Hub description.

**Comment by Quantisan** [comment]
@tianon got it. i've removed 2.5.0

**Comment by tianon** [comment]
:heart:

Build test of #436; 1c0f8c7af8643e8d495a6eef2d2f9c7df055983a (`clojure`):

```console
$ bashbrew build https://raw.githubusercontent.com/docker-library/official-images/1c0f8c7af8643e8d495a6eef2d2f9c7df055983a/library/clojure
Fetching clojure (git://github.com/Quantisan/docker-clojure) ...
Processing clojure:latest ...
Processing clojure:lein-2.5.1 ...
Processing clojure:lein-2.4.3 ...
```

**Comment by yosifkit** [comment]
LGTM

**Pull request merged**